src/main/scala/ai/starlake/schema/handlers/LocalStorageHandler.scala

```diff
@@ -147,7 +147,7 @@ class LocalStorageHandler(implicit
           else
             files.sortBy(f => (f.lastModifiedTime, f.name))
 
-        sorted.map(f => new Path(f.pathAsString))
+        sorted.map(f => new Path(f.uri))
```

Comment:
Does it make any difference ?

Reply:
it just add the protocol prefix. Tried to restore pathAsString and fix failed unit tests. It works as well and build on windows. Change reverted.